Classes/Repository/RepositoryCodeBlobViewController.swift

```diff
@@ -69,11 +81,25 @@ final class RepositoryCodeBlobViewController: UIViewController {
     }
 
     // MARK: Private API
+    
+    func didFetchPayload(_ payload: Any) {
+        sharingPayload = payload
+        sharingButton.isEnabled = true
+    }
 
     @objc
     func onRefresh() {
         fetch()
     }
+    
+    @objc
+    func onShare(sender: UIBarButtonItem) {
+        guard let payload = sharingPayload else { return }
+        let activityController = UIActivityViewController(activityItems: [payload], applicationActivities: nil)
```

Comment:
Should we just be sharing `textView.text` so we aren't duplicating the "source of truth" here?

Reply:
Check above. How would we deal with this if the “source” is an image?